frontend/lib/Repository/AdminRepository/admin_gateway.dart

```diff
@@ -29,4 +30,9 @@ class AdminGateway {
       restRepository.getEndpointsForGroup(groupId);
 
   Future<bool> deleteMember(int memberId, int groupId) => restRepository.deleteMember(memberId, groupId);
+
```

Comment:
Lepszym miejscem na to byłby plik `groups_repository`, tam mamy wszystko co związane z grupami, a te zapytania są pod jeną konkretną funkcjonalność